### ISSUE TITLE
Not working

### ISSUE THREAD

**Issue by ravicant** [pitaj/nodebb-plugin-forms#10]
It does not work anymore?

**Comment by yariplus** [comment]
I don't think it has worked for some time now.

Is there a specific feature you are looking for? I might be able to make or modify a plugin that does what you need.

Thanks!

**Comment by ravicant** [comment]
> I don't think it has worked for some time now.
> 
> Is there a specific feature you are looking for? I might be able to make or modify a plugin that does what you need.
> 
> Thanks!

yes i was looking to shift fully the most imp thing i need a form which i can let other people to fill and then it will make a post with question where certain group and the guy who filled can see and then we can let them know if the form is accepted or not rn i am using xenforo but its sucks ass soo much 

https://india.legacy-roleplay.com/index.php?form/

something like this https://img.ravicant.in/u/QPt3ei.mp4

if You can help will be huge help

**Comment by yariplus** [comment]
I'm looking over the old code, and it would be a lot of work to get it to the point it matches the video.

It could possibly build the form like you have posted, but the only actions it does when submitted is to make an email or a post with the static data. There is no tracking of the form after it is sent, or any controls on who can submit what form (any user can submit a ban appeal multiple times etc.). There also isn't any granular post permissions in NodeBB that I know of, so the post would not be viewable by the user.
